src/esp/gfx_batch/Renderer.h

```diff
@@ -640,13 +640,24 @@ class Renderer {
   void clearLights(Magnum::UnsignedInt sceneId);
 
   /**
-   * @brief Combined camera projection and transformation matrix
-   * @param sceneId   Scene ID, expected to be less than @ref sceneCount()
+   * @brief Get the combined projection and view matrices of a camera
+   * (read-only)
+   * @param sceneId Scene ID, expected to be less than @ref sceneCount()
+   */
+  const Magnum::Matrix4& camera(Magnum::UnsignedInt sceneId) const;
+
+  /**
+   * @brief Set the camera projection and view matrices
+   * @param sceneId Scene ID, expected to be less than @ref sceneCount()
+   * @param view View matrix of the camera (inverse transform)
+   * @param projection Projection matrix of the camera
    *
-   * Modifications to the transformation are taken into account in the next
-   * @ref draw().
+   * Also computes the camera unprojection.
+   * Modifications to the transformation are taken into account in the next @ref draw().
```

Comment:
Why not keeping the comment wrapped to 80 cols? :) Those are excluded from clang-format because newlines at certain positions inside `@ref` break Doxygen, but I like to have many editors side-by-side and there the comment wrapping is useful.